azure-pipelines.yml

```diff
@@ -117,3 +155,15 @@ stages:
                 inputs:
                   targetPath: artifact
                   artifact: keboola.job-runner.latest-build
+
+              - task: DownloadSecureFile@1
+                name: downloadPrivateKey
+                inputs:
+                  secureFile: "keboola-kbc-stacks-trigger-241022-pvk.pem"
+                displayName: 'Download "Keboola - kbc-stacks trigger" GitHub App Private Key for updating Helm charts'
```

Comment:
Tohle je predpokladam potreba pro ten help chart step? Nedavalo by smysl to zabalit do nej?

Reply:
Asi by to slo. Tady to tak je https://github.com/keboola/sapi-importer/blob/8fb23d852b0a6f20ede47aa4289ef3679edd4896/azure-pipelines/steps/update-helm-chart.yml#L15

Nevim proc u tech novejsich je to venku.